File: 1/main.pas
program f1;
var a, x, b, y: real;

function f (x, a, b: real): real;
  begin
  Result := ((b + Power(SIN(a * x),  2)) / (EXP(1) ** (-x / 2)))
  end;

begin
write('x = ');
readln(x);
write('a = ');
readln(a);
write('b = ');
readln(b);

y := f(x, a, b);
write('y = ');
writeln(y)
end.

File: 1/main.py
import math as m

def f(x: float, a: float, b: float) -> float:
    return (b + m.sin(a*x) ** 2) / (m.e ** (-x / 2))

x = float(input("x = "))
a = float(input("a = "))
b = float(input("b = "))

y = f(x, a, b)
print(f'y = {y}')
